blog/architecture-developer-platforms.md

```
@@ -0,0 +1,13 @@
+title: "Architecture of Developer Platforms"
+date: 2024-05-07
+tags: ["idp", "crossplane"]
+# The Architecture of Developer Platforms
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)